test/core/debug/debug-screens.test.js

```diff
@@ -50,6 +52,7 @@ describe("getDebugScreens", () => {
                 navigation: { "debug-mockKey1": "", "debug-mockKey2": "", "debug-mockKey3": "" },
                 screen: "mockConfig",
             });
+            expect(Load.loadCollections).toHaveBeenCalledWith(mockScreen, mockScreen.context.config, "debug/");
```

Comment:
couldn't separate this into another test, something weird going on, probably because the function `addExampleScreens` is a singleton?

Reply:
It's debug so prob not worth worrying about too much.